cabal-install/src/Distribution/Client/CmdOutdated.hs

```diff
@@ -161,10 +168,6 @@ outdatedOptions _showOrParseArgs =
       "Act on the new-style freeze file (default: cabal.project.freeze)"
       outdatedNewFreezeFile (\v flags -> flags {outdatedNewFreezeFile = v})
       trueArg
-  , option [] ["project-file"]
-      "Act on the new-style freeze file named PROJECTFILE.freeze rather than the default cabal.project.freeze"
```

Comment:
This will change the description to a slightly inaccurate "Set the name of the cabal.project file to search for in parent directories", no? This is particularly tricky for the user because the flag only sets the PROJECTFILE part of PROJECTFILE.freeze

Reply:
Hmm, though since this isn't the `freeze` command, one would expect --project-file to affect both PROJECTFILE and PROJECTFILE.freeze like the other commands (?). Is that what this flag does / will do?

Reply:
Yes it does!

Reply:
I agree, this way the flag is slightly less discoverable

Reply:
> Yes it does!

then it's accurate, though maybe we should improve the help text for --project-file, like in https://cabal.readthedocs.io/en/3.4/cabal-project.html#cmdoption-0